www/shared/test-utils/modules/react-app-tree/AppTree.test.js

```diff
@@ -1,38 +1,38 @@
 import React from 'react';
 import { FormattedMessage } from 'react-intl';
-import { render } from '@testing-library/react';
+import { render, screen } from '@testing-library/react';
 import AppTree from './AppTree';
 
 it('should render children correctly', () => {
-    const { container } = render(
+    render(
```

Comment:
Instead of using screen, shouldn’t we use the container returned by render?

Reply:
I saw your comment now. Maybe we should then be using screen in all other tests or you were having issues with it?

Reply:
I think I'm already using `screen` whatever it's possible to do it, but I'll double check the test files anyway.

Reply:
Maybe you find strange that `container` is still being used quite often. However, I can't replace it with `screen` if the assertion needs to use `querySelector` for instance or anything else from the DOM API.

Reply:
Just double checked and there are still cases where `container` isn't needed at all. Example: https://github.com/moxystudio/next-with-moxy/pull/101/files#diff-46c595aa0102a72440ca2a3bf2d1bd5a

Reply:
All good then?

Reply:
All good 🙌

Reply:
I've opened an [issue](https://github.com/moxystudio/eslint-config/issues/101) recently in our ESLint config to consider adding two new plugins for RTL. One of them has a [rule](https://github.com/testing-library/eslint-plugin-testing-library/blob/master/docs/rules/prefer-screen-queries.md) precisely enforce the usage of screen.

Reply:
> Maybe you find strange that `container` is still being used quite often. However, I can't replace it with `screen` if the assertion needs to use `querySelector` for instance or anything else from the DOM API.

True. However, perhaps we can create a few [custom queries](https://testing-library.com/docs/react-testing-library/setup#add-custom-queries) to find and select elements with custom attributes:

- `[data-wait-for-react-element="progressBar"]`
- `[data-wait-for-react-element="splashScreen"]`

Reply:
Good idea @acostalima! I will add a custom query for this case

Reply:
Done @acostalima

Reply:
Good to go! 🚀